tests/zenko_tests/configuration.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+
+import logging
+import os
+import sys
+import time
+
+import yaml
+import jwt
+from bravado.client import SwaggerClient
+from bravado.requests_client import RequestsClient
+from bravado_core.spec import Spec
+from kubernetes import client, config
+from kubernetes.config.config_exception import ConfigException
+from jsonschema import validate
+
+from e2e_config import accounts, endpoints, locations, workflows, schema
+
+logging.basicConfig(level=logging.INFO)
+_log = logging.getLogger("end2end configuration")
+
+
+def load_k8s_config(config_file=""):
+    """
+    Loads K8S config.
+
+    :param config_file: Name of the kube-config file.
+    """
+    try:
+        config.load_incluster_config()
+        return
+    except:
+        _log.info("unable to load incluster config")
+
+    try:
+        config.load_kube_config(config_file=config_file)
+        return
+    except:
+        _log.info("unable to load k8s config file")
+
+    raise Exception("Failed to load k8s config")
+
+
+def load_end2end_config(config_file=""):
+    """
+    Loads end-to-end test setup file
+    """
+    try:
+        with open(config_file) as file:
+            return yaml.full_load(file)
+    except:
+        raise Exception("Failed to load end2end config file")
```

Comment:
Not sure the benefit of catching an exception just to raise a generic one. If the yaml file was not found or is not a proper yaml file, full_load will already raise a very similar exception ( maybe more specific ).